spectrum-offchain-lm/src/funding.rs

```diff
@@ -44,22 +44,19 @@ where
     R: FundingRepo,
 {
     async fn collect(&mut self, target: NanoErg) -> Result<NonEmpty<AsBox<DistributionFunding>>, ()> {
-        trace!(target: "funding", "collect(target: {:?})", target);
```

Comment:
It's better to keep traces preceding inner call, then we will at least see which method was called last in case of deadlock or error

Reply:
Brought back the deleted traces.